src/tools/run-make-support/src/lib.rs

```diff
@@ -7,6 +7,8 @@
 // skip rustfmt on this file.
 #![cfg_attr(rustfmt, rustfmt::skip)]
 
+pub use serde_json;
+
```

Comment:
Suggestion: there is a smol section of pub re-exports of third-party libs just a few lines below, could we move it there?

https://github.com/rust-lang/rust/blob/7116ed4983119b6506d954c6bad64f07b3017c83/src/tools/run-make-support/src/lib.rs#L38

Reply:
Ah sure!